client/signup/main.jsx

```diff
@@ -125,6 +125,10 @@ function isWPForTeamsFlow( flowName ) {
 	return flowName === 'p2';
 }
 
+function isReskinnedFlow( flowName ) {
+	return config.isEnabled( 'signup/reskin' ) && config( 'reskinned_flows' ).includes( flowName );
+}
```

Comment:
Love this.

Reply:
Since it's not an experiment anymore, it's about time to find a somewhat scalable solution. When all the signup flows would be 'reskinned', we can clean all of these but I feel that we're going to live with blue and white for some time.